assets/js/romo/sortable.js

```diff
@@ -96,54 +98,59 @@ RomoSortable.prototype.onDragStart = function(e) {
 
-  this.draggedIndex = this.draggedElem.index();
+  var elems = Romo.children(Romo.parent(this.draggedElem));
+  this.draggedIndex = elems.indexOf(this.draggedElem);
 
-  this.placeholderElem.css({ 'height': this.draggedElem.height() });
+  Romo.setStyle(this.placeholderElem, 'height', Romo.css(this.draggedElem, 'height'));
 
-  this.elem.trigger('sortable:dragStart', [this.draggedElem, this]);
+  Romo.trigger(this.elem, 'romoSortable:dragStart', [this.draggedElem, this]);
 }
 
 RomoSortable.prototype.onDragEnter = function(e) {
   e.preventDefault();
   e.stopPropagation();
 
   // return if event is fired on the placeholder
-  if(this.placeholderElem.get(0) === e.currentTarget){ return; }
+  if(this.placeholderElem === e.currentTarget){ return; }
 
-  this.placeholderElem.show();
-  this.draggedElem.hide();
+  Romo.show(this.placeholderElem);
+  Romo.hide(this.draggedElem);
 
   // if event is not fired on the sortable
-  var overSortableElem = this.elem.get(0) === e.currentTarget;
-  var clientX = e.originalEvent.clientX;
-  var clientY = e.originalEvent.clientY;
+  var overSortableElem = this.elem === e.currentTarget;
+  var clientX = e.clientX;
+  var clientY = e.clientY;
   if (!overSortableElem) {
     // if we are in the same elem and moving the same direction, exit out
     var overSameElem = this.draggedOverElem &&
-                       this.draggedOverElem.get(0) === e.currentTarget;
+                       this.draggedOverElem === e.currentTarget;
     var sameDirection = (this.dragDirection === 'down' && clientY > this.lastY) ||
                         (this.dragDirection === 'up'   && clientY < this.lastY);
     if(overSameElem && sameDirection){ return; }
 
     // remove dragged over classes from previous elem
-    if(this.draggedOverElem){ this.draggedOverElem.removeClass(this.dragOverClass); }
-    this.draggedOverElem = $(e.currentTarget);
+    if(this.draggedOverElem){ Romo.removeClass(this.draggedOverElem, this.dragOverClass); }
+    this.draggedOverElem = e.currentTarget;
     this.lastY = clientY;
-    this.draggedOverElem.addClass(this.dragOverClass);
+    Romo.addClass(this.draggedOverElem, this.dragOverClass);
 
     // insert the placeholder according to the dragging direction
-    if (this.placeholderElem.index() < this.draggedOverElem.index()) {
+    var elems = Romo.children(Romo.parent(this.placeholderElem));
+    var placeholderIndex = elems.indexOf(this.placeholderElem);
+    elems = Romo.children(Romo.parent(this.draggedOverElem));
+    var draggedOverIndex = elems.indexOf(this.draggedOverElem);
+    if (placeholderIndex < draggedOverIndex) {
       this.dragDirection = 'down';
     } else {
       this.dragDirection = 'up';
     }
     var insertMethod = this.dragDirection === 'down' ? 'after' : 'before';
-    this.draggedOverElem[insertMethod](this.placeholderElem);
+    Romo[insertMethod](this.draggedOverElem, this.placeholderElem);
```

Comment:
@jcredding what is `Romo[insertMethod]` here?

Reply:
@kellyredding - You can access functions/attributes/values whatever you want to call them via `[]` on javascript objects. `Romo` is just a javascript object so `Romo['after']` or `Romo['before']` is the same as `Romo.after` and `Romo.before`.

Reply:
@jcredding ahh yes, that's right.  Thanks for clarifying this man.